src/compiler/types.ts

```diff
@@ -693,6 +693,8 @@ namespace ts {
         typeParameters?: NodeArray<TypeParameterDeclaration>;
         parameters: NodeArray<ParameterDeclaration>;
         type: TypeNode | undefined;
+        /* @internal */
+        typeArguments?: NodeArray<TypeNode>; // Used for quick info, replaces typeParameters for instantiated signatures

```

Comment:
nit: put `@internal` annotation on the same line.

Reply:
done